setup.py

```diff
@@ -53,7 +53,7 @@ def read(fname):
     license="Apache-2.0",
     test_suite="test",
     entry_points={
-        "console_scripts": ["gt=greentea.greentea_cli:main", "htrun=htrun.htrun:main"],
+        "console_scripts": ["htrun=htrun.htrun:main"],
```

Comment:
Do we want to consider changing the command line to `greentea-host` or do we want to leave it as `htrun`?

Reply:
IMO We go with `htrun` as it is actually a host tests runner

Reply:
Fair enough.